content/blog/2016-09-02-promcon-2016-its-a-wrap.md

```diff
@@ -0,0 +1,152 @@
+their systems using Prometheus. [Weaveworks](https://www.weave.works/)
+explained how they built a scalable multi-tenant version of Prometheus.
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">.<a href="https://twitter.com/cagedmantis">@cagedmantis</a> about how they are deploying <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> at <a href="https://twitter.com/digitalocean">@digitalocean</a> <a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> <a href="https://t.co/fc7yggVtyY">pic.twitter.com/fc7yggVtyY</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/769161605045161988">August 26, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">.<a href="https://twitter.com/Carretops">@Carretops</a> from <a href="https://twitter.com/ShuttleCloud">@ShuttleCloud</a> is explaining how <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> works well for small companies <a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> <a href="https://t.co/5ccvGY4fQy">pic.twitter.com/5ccvGY4fQy</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/768724543062024192">August 25, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">Directly from Japan, <a href="https://twitter.com/wyukawa">@wyukawa</a> from <a href="https://twitter.com/LINEjp_official">@LINEjp_official</a> about Hadoop/Fluentd monitoring with <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> <a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> <a href="https://t.co/irJswioxMf">pic.twitter.com/irJswioxMf</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/769171613577310208">August 26, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet tw-align-center" data-lang="en"><p lang="en" dir="ltr">It&#39;s getting spooky! <a href="https://twitter.com/tom_wilkie">@tom_wilkie</a> from <a href="https://twitter.com/weaveworks">@weaveworks</a> introduces Frankenstein, a distributed Prometheus. <a href="https://twitter.com/hashtag/promcon2016?src=hash">#promcon2016</a> <a href="https://t.co/6xgvlYv6Tw">pic.twitter.com/6xgvlYv6Tw</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/768826444575309824">August 25, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Several Prometheus core developers also talked about the the design decisions
```

Comment:
I could add that... just felt I was pushing the limit with respect to how many Twitter cards the post could accomodate. Maybe just add it in the text here?